examples/post-processing/rs-post-processing.cpp

```diff
@@ -269,11 +272,29 @@ void render_ui(float w, float h, std::vector<filter_options>& filters)
         ImGui::Checkbox(filter.filter_name.c_str(), &tmp_value);
         filter.is_enabled = tmp_value;
         ImGui::PopStyleColor();
+        
+         
+        if( filter.filter_name == "Rotate" )  // Combo box specifically for the rotation filter
```

Comment:
I wonder why rotate is the only one needs special handling,
Is it because we added a special option for it?
What is the value of RS2_OPTION_FILTER_MAGNITUDE for other filters?
Is it always 1,2,3,4 ?
If not maybe we could have used it for 90/-90/180?

Reply:
Because the rest of the options use filter_slider_ui which is set up for continuous values, but we need discrete 90-degree increments.